addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -12,10 +12,47 @@
     html {
         height: unset;
     }
+    @page {
+        size: portrait; /* force paper orientation to portrait */
+    }
+    canvas {
+        margin-bottom: 2rem;
+    }
+    .o_survey_question_page {
+        page-break-inside: avoid;
+    }
+    .o_survey_results_question_wrapper {
+        .o_survey_results_question_header, .o_survey_question_description {
+            page-break-inside: avoid;
+            page-break-after: avoid;
+        }
+    }
+    .o_survey_results_question_wrapper:has(button.collapsed) {
+        display: none;
+    }
+    .o_survey_user_responses_table_wrapper {
+        height: auto !important;
+    }
+    .o_survey_brand_message {
+        border: none !important;
+    }
+    table {
```

Comment:
Same remark here, seems it could impact all 'table'; edit valid for other global changes in this file, seems a bit louche :)